apache/ozone
# HDDS-7114. CHAINED and STAND_ALONE replication-type should be removed from help message

## What changes were proposed in this pull request?

CHAINED and STAND_ALONE replication-type should be removed from help message and other places in codes.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-7114

## How was this patch tested?
Manual tests via testing command 
`ozone sh bucket create /s3v/qe-dataconn-buck1 --type=ec --replication=rs-3-2-1511k`

After the code changes, the error message will remove STAND_ALONE and CHAINED type from CLI's help message
`Invalid value for option '--replication-type': expected one of [RATIS, STAND_ALONE, CHAINED, EC] (case-sensitive) but was 'ec'`

to
`Could not invoke public void org.apache.hadoop.ozone.shell.ShellReplicationOptions.setType(java.lang.String) with ec (java.lang.IllegalArgumentException: Invalid value 'ec' for option '--type'. No enum constant org.apache.hadoop.hdds.client.ReplicationType.ec. Supported values are: RATIS, EC.)`
